startups.csv

```diff
@@ -57,6 +57,7 @@ d-Matrix,d-matrix.ai,AI,US,2019,Computing platform for data center AI inference
 DeGirum,degirum.ai,AI,US,2017,AI hardware and software to accelerate AI edge adoption
 Denpaflux,denpaflux.com,EDA,DE,2022,AI based PCB design
 Dfiant,dfiant.works,EDA,US,2022,Chip design tools
+Dnotitia,dnotitia.com,AI,KR,2023,Vector DPUs, LLM systems, and HW/SW solutions for generative AI and AGI.
```

Comment:
... you need quotes or some other escape mechanism for commas in the Comma Separated Values format.